core/lib/server/email.tsx

```diff
@@ -113,6 +111,36 @@ export function passwordReset(
 	});
 }
 
+export function verifyEmail(
+	user: Pick<Users, "id" | "email" | "firstName" | "lastName">,
+	redirectTo?: string,
+	trx = db
+) {
+	return buildSend(async () => {
+		const magicLink = await createMagicLink(
+			{
+				type: AuthTokenType.generic,
```

Comment:
could you maybe add a small comment here why this needs to be `AuthTokenType.generic` vs `AuthTokenType.verifyEmail`? I think it's bc `/magic-link` will set their email to be verified, but i think that's easy to forget!